pennylane/circuit_drawer/__init__.py

```diff
@@ -19,8 +19,8 @@
 .. autosummary::
     :toctree: api
 
-    circuit_drawer
-    charsets
+    MPLDrawer
```

Comment:
Minor, but in the rendered documentation, there are two copies of `MPLDrawer`. I think it makes sense to just have the second one here?

![image](https://user-images.githubusercontent.com/2068515/132056951-4901d344-64a8-45ad-bfca-ffac88dd2c86.png)